creator-node/src/resizeImage.js

```diff
@@ -112,15 +111,15 @@ module.exports = async (job) => {
     square,
     logContext
   } = job.data
-  const ipfs = ipfsClient(
-    config.get('ipfsHost'),
-    config.get('ipfsPort')
-  )
-
   const logger = genericLogger.child(logContext)
 
   // Read the image once, clone it later on
-  let img = await Jimp.read(file)
+  let img
+  try {
+    img = await Jimp.read(file)
+  } catch (e) {
+    throw new Error(`Could not generate image buffer during image resize: ${e}`)
```

Comment:
what's the benefit of catching this error if we just always re-throw it with additional msg?

i'm not familiar with this codepath but i don't see any logic in `ImageProcessingQueue.js` to handle this error - looks like if thrown the queue task would still just die?

Reply:
ah that's a good point; I wanted to add this error for more specificity on what went wrong, but taking a closer look, the queue would just fail upon this error. @raymondjacobson we should also add a try/catch to the `ImageProcessingQueue` process like you did for `TranscodingQueue`!

Reply:
actually, now that I take a second look at this, `ImageProcessingQueue` gets used in `files.js`, and that is wrapped in a try/catch. the specific error should be caught